src/views/virtualmachines/migrate/VirtualMachineMigration.tsx

```diff
@@ -0,0 +1,184 @@
+          groupVersionKind: VirtualMachineModelGroupVersionKind,
+          name,
+          namespace,
+        }
+      : null,
+  );
+
+  const { pvcs } = useDisksSources(vm);
+
+  const [{ clusterDefaultStorageClass, sortedStorageClasses }, scLoaded] = useDefaultStorageClass();
+
+  const defaultStorageClassName = getName(clusterDefaultStorageClass);
+
+  const destinationStorageClass = selectedStorageClass || defaultStorageClassName;
+
+  const onSubmit = async () => {
+    setMigrationLoading(true);
+    setMigrationError(null);
+    try {
+      await migrateVM(vm, pvcs, destinationStorageClass);
+
+      navigate(-1);
+    } catch (apiError) {
+      setMigrationError(apiError);
+    }
+
+    setMigrationLoading(false);
+  };
+
+  if (!loaded && !scLoaded)
+    return (
+      <Bullseye>
+        <Loading />
+      </Bullseye>
+    );
+
+  if (error)
+    return (
+      <Alert isInline title={t('Error')} variant={AlertVariant.danger}>
+        {error?.message}
+      </Alert>
+    );
+
+  return (
+    <Wizard
+      header={
+        <WizardHeader
+          closeButtonAriaLabel={t('Close header')}
+          description={t('Migrate VirtualMachine storage to a different StorageClass.')}
+          title={t('Migrate VirtualMachine storage')}
+        />
+      }
+      onClose={() => navigate(-1)}
+      onSave={onSubmit}
+      title={t('Migrate VirtualMachine storage')}
+    >
+      <WizardStep id="wizard-migrate-destination" name={t('Destination StorageClass')}>
+        <Stack hasGutter>
+          <StackItem>
+            <Title headingLevel="h2">{t('Destination StorageClass')}</Title>
+            <p>{t('Select the destination storage for the VirtualMachine storage migration.')}</p>
+          </StackItem>
+          <StackItem>
+            <InlineFilterSelect
+              options={sortedStorageClasses?.map((storageClass) => ({
+                children: (
+                  <>
+                    <ResourceLink
+                      groupVersionKind={StorageClassModelGroupVersionKind}
+                      inline
+                      linkTo={false}
+                      name={storageClass}
+                    />
+                    {defaultStorageClassName === storageClass ? '(default)' : ''}
+                  </>
+                ),
+                value: storageClass,
+              }))}
+              selected={destinationStorageClass}
+              setSelected={setSelectedStorageClass}
+              toggleProps={{ isFullWidth: true, placeholder: t('Select StorageClass') }}
+            />
+          </StackItem>
+        </Stack>
+      </WizardStep>
+      <WizardStep
+        footer={{
+          isNextDisabled: migrationLoading,
+          nextButtonText: t('Migrate VirtualMachine storage'),
+        }}
+        id="wizard-migrate-review"
+        name={t('Review')}
+      >
+        <Stack hasGutter>
+          <StackItem>
+            <Title headingLevel="h2">{t('Review')}</Title>
+            <p>
```

Comment:
You can use Text/Typography from pattternfly; it better because it aimed to font and size of p across all PF components